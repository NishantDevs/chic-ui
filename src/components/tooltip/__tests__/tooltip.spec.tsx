import Tooltip from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Tooltip Component', () => {
    it('Renders Correctly', () => {
        const TooltipGroup = render(
        <>
            <Tooltip placement='top' content='Tooltip Top' className='tooltip top' />
            <Tooltip placement='left' content='Tooltip Left' className='tooltip left' />
            <Tooltip placement='bottom' content='Tooltip Bottom' className='tooltip bottom' />
            <Tooltip placement='right' content='Tooltip Right' className='tooltip right' />
        </>
        );
        expect(TooltipGroup).toMatchSnapshot();
    });
});
