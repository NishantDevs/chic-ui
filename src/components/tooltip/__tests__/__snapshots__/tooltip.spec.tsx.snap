// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip Component Renders Correctly 1`] = `
Array [
  <button
    aria-label="Tooltip Top"
    className="sc-bdfBQB euoskl tooltip top"
    type="button"
  >
    Tooltip on 
    top
  </button>,
  <button
    aria-label="Tooltip Left"
    className="sc-bdfBQB bMeiSC tooltip left"
    type="button"
  >
    Tooltip on 
    left
  </button>,
  <button
    aria-label="Tooltip Bottom"
    className="sc-bdfBQB rtynV tooltip bottom"
    type="button"
  >
    Tooltip on 
    bottom
  </button>,
  <button
    aria-label="Tooltip Right"
    className="sc-bdfBQB kYJiyS tooltip right"
    type="button"
  >
    Tooltip on 
    right
  </button>,
]
`;
